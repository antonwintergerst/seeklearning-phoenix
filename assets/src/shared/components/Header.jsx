import React from 'react';

const Header = () => (
  <header className="header" role="banner">
    <div className="business-menu">
      <div className="ui container">
        <nav className="ui menu">
          <a className="item">Jobs</a>
          <a className="item active">Courses</a>
          <a className="item">Businesses for sale</a>
          <a className="item">Volunteering</a>
        </nav>
      </div>
    </div>
    <div className="ui logo container">
      <img src="/dist/images/seeklearning-logo.svg" alt="Seek Learning" />
    </div>
  </header>
);

export default Header;
