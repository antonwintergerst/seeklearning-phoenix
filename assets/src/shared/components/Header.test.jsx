import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import Header from './Header';

describe('Header', () => {
  it('renders as expected', () => {
    const tree = ReactTestRenderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
