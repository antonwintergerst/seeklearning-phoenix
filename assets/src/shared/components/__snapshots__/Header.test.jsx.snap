// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders as expected 1`] = `
<header
  className="header"
  role="banner"
>
  <div
    className="business-menu"
  >
    <div
      className="ui container"
    >
      <nav
        className="ui menu"
      >
        <a
          className="item"
        >
          Jobs
        </a>
        <a
          className="item active"
        >
          Courses
        </a>
        <a
          className="item"
        >
          Businesses for sale
        </a>
        <a
          className="item"
        >
          Volunteering
        </a>
      </nav>
    </div>
  </div>
  <div
    className="ui logo container"
  >
    <img
      alt="Seek Learning"
      src="/dist/images/seeklearning-logo.svg"
    />
  </div>
</header>
`;
